packages/components/test/pseudolocalize.test.js

```diff
@@ -1,10 +1,9 @@
-import { assert } from 'chai';
 import { pseudolocalize } from '../src';
 
 describe('pseudolocalize', () => {
-  it('should transform a message', () => {
+  it.only('should transform a message', () => {
```

Comment:
careful about leaving these in.

Is there a linting rule or similar to prevent accidentally merging this in to master?